README.md

```diff
@@ -41,6 +41,16 @@ Some applications load from `.env`, `.env.local`, `.env.development` and `.env.d
 (see [#37](https://github.com/entropitor/dotenv-cli/issues/37) for more information).
 `dotenv-cli` supports this using the `-c` flag for just `.env` and `.env.local` and `-c development` for the ones above.
 
+### Override
+Override any environment variables that have already been set on your machine (or in case of using multiple environment files, defined by previous environment file) with values from your .env file.
```

Comment:
I think this is very strange and will lead to weird behavior where adding `--override` not only ignores env variables already set, it also changes the order of `-e` arguments.

`--override` should in my opinion NOT reverse the order in which you should declare them. It should only decide whether it uses the system ones or not.

Reply:
The current state of this PR reflects my misunderstanding of how the order of `-e` arguments impact environment variables applied to the child process. 

Later today, in a follow-up commit, I will change the language of this documentation and the help command to reflect that the purpose of `--override` is to override an existing environment variable and remove the language around using it to override environment variables defined by other `.env` files.

Reply:
Updated the docs in 9124e50.